src/StatsdClient/StatsdClient.csproj

```diff
@@ -16,9 +16,10 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(OS)' == 'Windows_NT' ">
-    <DefineConstants>OS_WINDOWS</DefineConstants>
```

Comment:
we still use the `OS_WINDOWS` define in a few places, should we really remove this?